akka-stream-tests/src/test/scala/akka/stream/io/InputStreamSinkSpec.scala

```diff
@@ -219,4 +220,11 @@ class InputStreamSinkSpec extends AkkaSpec(UnboundedMailboxConfig) {
       inputStream.close()
     }
   }
+
+  "fail to materialize with zero sized input buffer" in {
+    an[IllegalArgumentException] shouldBe thrownBy {
+      Source.single(byteString)
```

Comment:
I used `Source.single` instead of `Source.empty`, because zero sized buffer error would be picked up even without this fix.